src/main/java/nextstep/subway/applicaion/strategy/strategy/PathFindStrategy.java

```diff
@@ -0,0 +1,13 @@
+package nextstep.subway.applicaion.strategy.strategy;
+
+import nextstep.subway.domain.Station;
+
+import java.util.List;
+
+public interface PathFindStrategy {
+
+    List<Station> findShortPath(Station source, Station target);
+    int getShortestDistance(Station source, Station target);
```

Comment:
```suggestion
    Path findShortPath(Station source, Station target);
```
으로도 만들 수 있을지도 모릅니다 ㅎㅎ 
DijkstraPathFindStrategy 내부에 인스턴스 변수로 정점과 간선을 가지고 있기보다는
PathFindCondition이라는 객체를 하나 만들어서 빌더패턴으로 작성해볼수도 있죠 ㅎㅎ

```java
Path path = PathFinder.condition()
        .vertextList(stations)
        .edgeList(lines)
        .startPoint(startStation)
        .endPoint(endStation)
        .search();
```

이런식으로도 말이죠 ㅎㅎ